src/schemaTypes.ts

```diff
@@ -25,7 +25,7 @@ export interface loginVariables {
 // GraphQL query operation: CheckUserToken
 // ====================================================
 
-export interface CheckUserToken {
+export interface ICheckUserToken {
```

Comment:
You should not change anything on this file, cause it is automatically generated.